wit/graphics-context.wit

```diff
@@ -1,36 +1,14 @@
 package wasi:webgpu;
 
 interface graphics-context {
-
-    resource graphics-context {
+    resource context {
         constructor();
 
-        configure: func(desc: configure-context-desc);
-
-        get-current-buffer: func() -> graphics-context-buffer;
+        get-current-buffer: func() -> buffer;
 
+        // TODO: might want to remove this.
         present: func();
     }
 
-    // might not make sense here
-    record configure-context-desc {
-        alpha-mode: composite-alpha-mode,
-        // colorSpace
-        // device?
-        // usage?
-        // format?
-        // viewFormats?
-        // getPreferredCanvasFormat?
-    }
-
-    resource graphics-context-buffer {
-    }
-
-    enum composite-alpha-mode {
-        auto,
-        opaque,
-        pre-multiplied,
-        post-multiplied,
-        inherit,
-    }
+    resource buffer { }
```

Comment:
How is this buffer used? I see how the equivalent in webgpu.wit and frame-buffer.wit are used.

Reply:
Can be converted into a gpu-texture in webgpu, or into a simple buffer in frame-buffer.
Maybe we should name it something like abstract-buffer?

Reply:
Makes sense - let's go with `abstract-buffer`